www/pages/stop/stop-controller.js

```diff
@@ -207,11 +229,14 @@ angular.module('pvta.controllers').controller('StopController', function ($scope
 
   // **Sets** whether a route's
   // departures have been expanded on the page
-  $scope.toggleGroup = function (group) {
-    if ($scope.isGroupShown(group)) {
+  $scope.toggleGroup = function (routeId, numberOfDepartures) {
+    if (numberOfDepartures === 1 && !_.contains($scope.onlyOneDeparture, routeId)) {
+      $scope.onlyOneDeparture.push(routeId);
+    }
```

Comment:
Since we're adding the first departure to the expanded view, "no more upcoming departures" is no longer necessary.  If a route has no departures, it simply won't be shown.